docs/Overview.md

```diff
@@ -19,6 +19,7 @@ Examples for each backend:
   * `leveldb`:  `./cayley init --db=leveldb --dbpath=/tmp/moviedb` -- where /tmp/moviedb is the path you'd like to store your data.
   * `bolt`:  `./cayley init --db=bolt --dbpath=/tmp/moviedb` -- where /tmp/moviedb is the filename where you'd like to store your data.
   * `mongo`: `./cayley init --db=mongo --dbpath="<HOSTNAME>:<PORT>"` -- where HOSTNAME and PORT point to your Mongo instance.
+  * `sql`: `./cayley init --db=sql --dbpath="user=user dbname=mydb password=mypassword sslmode=disable"` -- where the value of the `dbpath` flag is a connection string of parameters from [pq](https://github.com/lib/pq). See https://godoc.org/github.com/lib/pq for more information.
```

Comment:
Is `sslmode=disable` necessary, strictly speaking? Have you tried it with ssl enabled?